fem/hyperbolic_conservation_laws.cpp

```diff
@@ -0,0 +1,492 @@
+// Copyright (c) 2010-2023, Lawrence Livermore National Security, LLC. Produced
+// at the Lawrence Livermore National Laboratory. All Rights reserved. See files
+// LICENSE and NOTICE for details. LLNL-CODE-806117.
+//
+// This file is part of the MFEM library. For more information and source code
+// availability visit https://mfem.org.
+//
+// MFEM is free software; you can redistribute it and/or modify it under the
+// terms of the BSD-3 license. We welcome feedback and contributions, see file
+// CONTRIBUTING.md for details.
+
+// Implementation of hyperbolic conservation laws
+
+#include "hyperbolic_conservation_laws.hpp"
+#ifdef MFEM_USE_MPI
+#include "pnonlinearform.hpp"
+#endif
```

Comment:
```suggestion
#include "pnonlinearform.hpp"
```

There are already MPI `#ifdef`s in `pnonlinearform.hpp`, so they are not required here.